### PR TITLE
change the text in the HUD to images

### PR DESCRIPTION
**Pull request by noahbei**
Energy bar is based off of game.engery which is out of 6 just like the health. will need logic to make it work. can be altered in the future but it is out of 6 for now because that is how many bars are in the image.

### CODE REVIEW

File: UI.js [hunk 1]
```diff
@@ -14,11 +16,47 @@ draw(context){
...
+        }
+        else {
+            yDist = 0;
+            barX = 0;
+        }
+        const barWidth = 32;
+        let frameNum = Math.max(0, 6 - this.game[type]);
+        let frame = frameNum * barWidth;
+        if (this.game[type] < 6) frame-= barWidth;
+        context.drawImage(images.bars, 64 + frame, yDist, 31, 18, barX, -14, 96, 60);
```
Comment (JeremiahHerring) [inline UI.js:54]: TEST

File: UI.js [hunk 2]
```diff
@@ -14,11 +16,47 @@ draw(context){
...
     // score
-    context.fillText('Points: ' + this.game.score, 800, 35) // .fillText() - draws filled text on the canvas
-    context.fillText('Hedgehog: ' + this.game.hedgehogScore, 550, 35);
-    context.fillText('Bee: ' + this.game.beeScore, 405, 35);
-    context.fillText('Health: ' + this.game.health, 210, 35);
+    const scoreXOffset = 785;
+    context.fillText('Points: ' + this.game.score, scoreXOffset, 35) // .fillText() - draws filled text on the canvas
+
+    // draw hedgehog count
+    const hedgehogCountXOffset = 650;
```
Comment (JeremiahHerring) [inline UI.js:25]: test

File: main.js [hunk 3]
```diff
@@ -23,7 +23,8 @@ const imageArr = [
     'hedgehogImage.png',
     'fireEffect.gif',
     'gravityEffect.gif',
-    'boomEffect.png'
+    'boomEffect.png',
+    'bars.png'
```
Comment (JeremiahHerring) [inline main.js:27]: test

File: UI.js (hunk 1 above)
Comment (noahbei) [reply UI.js:54]: nice

**Pull request merged**